cmd/networkcmd/helpers.go

```diff
@@ -63,11 +63,13 @@ func TrackSubnet(
 	}
 
 	pluginPath := filepath.Join(app.GetPluginsDir(), vmID.String())
-	if err := utils.FileCopy(vmBin, pluginPath); err != nil {
-		return err
-	}
-	if err := os.Chmod(pluginPath, constants.DefaultPerms755); err != nil {
-		return err
+	if !utils.IsExecutable(pluginPath) {
```

Comment:
Should we be checking if the file exists before creating a new copy, and then checking if it's not already executable before doing the chmod?

Reply:
If the file exists and is not executable it is sign of a problem in the previous setup. It can be a partial copy for
example. So probably it is best to copy it again. 
Also, if it is not executable, there is not risk of encountering `text file busy` error because that happens
when the binary is being executed.

Reply:
Gotcha, could you add a comment explaining that context here? 🙏